pages/user guides/tidal-vms.md

```diff
@@ -62,9 +64,11 @@ If you need any help in deploying the VM or getting started with your discovery,
 
 ## Windows Server 2022
 
-The image runs [Windows Server (2022 Datacenter Evaluation)](https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2022). It comes only with the latest version of [Tidal Tools](https://tidalmigrations.com/tidal-tools/) and [Machine Stats](https://pypi.org/project/machine-stats/) installed. The Windows image does not include all the tools that are included in the Linux image.
+The image runs [Windows Server (2022 Datacenter Evaluation)](https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2022) and comes with a 180-day trial. It can be deployed into any virtual environment that is compatible with OVA based images.
 
-It can be deployed into any virtual environment that is compatible with OVA based images.
+{% include note.html content="This image comes only with the latest version of
+[Tidal Tools](https://tidalmigrations.com/tidal-tools/) and [Machine Stats](https://pypi.org/project/machine-stats/)
+installed. The Windows image does not consist all the tools and software that are included in the Ubuntu Server image." %}
```

Comment:
```suggestion
installed. The Windows image does not contain all the tools and software that are included in the Ubuntu Server image, since it is only intended to be used when capturing metrics with Machine Stats on a Windows environment" %}
```